packages/typescript-estree/tests/ast-alignment/fixtures-to-test.ts

```diff
@@ -364,46 +342,32 @@ tester.addFixturePatternConfig('typescript/basics', {
     'abstract-class-with-optional-method', // babel parse errors
     'declare-class-with-optional-method', // babel parse errors
     /**
-     * PR for parameter property ranges has been merged into Babel: https://github.com/babel/babel/pull/9284
-     * TODO: remove me in next babel > 7.2.3
+     * Was expected to be fixed by PR into Babel: https://github.com/babel/babel/pull/9284
+     * But not fixed in Babel 7.3
+     * TODO: Investigate differences
```

Comment:
looks like we have issue with ranges, `TSParameterProperty`->`AssignmentPattern`
`AssignmentPattern` includes modifier into their range